cmd/mp3togo/main.go

```diff
@@ -71,6 +82,25 @@ func (a *app) downloadVideo() {
 	defer file.Close()
 
 	vid.Download(best, file)
+	a.toConvert <- filename
+	go a.convertVideo(vid.Title)
+	a.wg.Add(1)
+}
+
+func (a *app) convertVideo(vidName string) {
+	defer a.wg.Done()
+
+	filename, open := <-a.toConvert
+	if !open {
+		fmt.Println("Channel closed")
+		return
+	}
+
+	newName := vidName + ".mp3"
+	a.ffmpeg.Args = append(a.ffmpeg.Args, filename, newName)
+
+	out, err := a.ffmpeg.Output()
```

Comment:
Shouldn't edit the value of `a.ffmpeg.Args` here. Makes more sense to use the temporary value returned by `append` for each video.
```suggestion
	out, err := append(a.ffmpeg.Args, filename, newName).Output()
```